common/src/main/java/com/wynntils/models/lootrun/LootrunModel.java

```diff
@@ -83,6 +103,67 @@ public void onChatMessage(ChatMessageReceivedEvent event) {
     public void onWorldStateChanged(WorldStateEvent event) {
         lootrunCompletedBuilder = null;
         lootrunFailedBuilder = null;
+
+        // FIXME: Persist this in a later PR.
```

Comment:
I tried persisting but it is a bit harder than it looks.. So I will hold off on doing that for now. I will come back to it soon.